packages/js-drive/lib/core/updateSimplifiedMasternodeListFactory.js

```diff
@@ -96,7 +97,11 @@ function updateSimplifiedMasternodeListFactory(
       contextLogger.debug(`SML is updated for core heights ${latestRequestedHeight} to ${coreHeight}`);
```

Comment:
In the first condition, we need `result` true too. Rename `result` to `isUpdated`